lib/polarshift/test_suite.rb

```diff
@@ -107,10 +107,8 @@ def commit!(test_case)
 
         if self.current_test_record.start_scenario_for! test_case
           return true
-        else
-          puts "logic error: starting test case '#{test_case}' that is not part of current test record\n"
```

Comment:
The change here is because we already have those statement info in method `start_scenario_for`
```
      def start_scenario_for!(test_case)
        scenario = self.test_case.scenarios.find { |s| s.match! test_case}
        unless scenario
          logger.warn "logic error: trying to start Cucumber scenario not part of this test record"
          logger.warn "Skipping testcase #{test_case}: #{test_case.name}"
        else
          scenario.start!
        end
      end
```